src/metabase/models/database.clj

```diff
@@ -434,14 +444,18 @@
 (defmethod serdes/make-spec "Database"
   [_model-name {:keys [include-database-secrets]}]
   {:copy      [:auto_run_queries :cache_field_values_schedule :caveats :dbms_version
-               :description :engine :is_audit :is_full_sync :is_on_demand :is_sample :metadata_sync_schedule :name
+               :description :engine :is_audit :is_attached_dwh :is_full_sync :is_on_demand :is_sample :metadata_sync_schedule :name
                :points_of_interest :refingerprint :settings :timezone :uploads_enabled :uploads_schema_name
                :uploads_table_prefix]
    :skip      [;; deprecated field
                :cache_ttl]
    :transform {:created_at          (serdes/date)
                ;; details should be imported if available regardless of options
-               :details             {:export #(if include-database-secrets % ::serdes/skip) :import identity}
+               :details             {:export (fn [{:keys [is_attached_dwh] :as db}]
```

Comment:
The argument coming to this function will be the value of `:details`, not the entire record. You would need to use the dynamic variable `serdes/*currrent* to read `is_attached_dwh` from.

Reply:
Thanks! https://github.com/metabase/metabase/pull/47247/commits/072fb725ee1392eb5b59382290eebc39ce1df60c